### ISSUE TITLE
Suggestion: support for direct pfam HMMER HMMSCAN output

### ISSUE THREAD

**Issue by reskejak** [kvittingseerup/IsoformSwitchAnalyzeR#20]
HMMER HMMSCAN is a more streamlined tool than the perl script pfam_scan.pl. To my knowledge, there is no way to output HMMER HMMSCAN in an identical format as pfam_scan.pl. Furthermore, the HMMSCAN webtool is constatly flooded with job requests resulting in long wait times (>12 hours). Support for direct HMMER output as Pfam input to IsoformSwitchAnalyzer would be useful to avoid dealing with finnicky perl scripts and dependencies.

**Comment by kvittingseerup** [comment]
That sounds like a really good idea! 

I have not tried using hmmscan before. One attractive feature of the pfam_scan.pl is that it only output significant features without the user having to specify any input (especially E-value cutoffs are trickey for non-expert users). Do you know how hmmscan compares?

**Comment by reskejak** [comment]
`hmmscan --tblout HMMSCAN_tblout_results.txt --acc Pfam-A.hmm isoformSwitchAnalyzeR_isoform_AA.fasta` should produce something similar to pfam_scan.pl output. I'm not familiar enough with the nuts and bolts of HMMER to know whether or not it should be exactly comparable by simple parsing, but you may wish to try toying with hmmscan.

**Comment by kvittingseerup** [comment]
I just tried it on my local installation and the main problem is the position where in the transcript a domain was found is not returned in any of the tabular formats (which is necessary for the plotting).  This means you have to parse the (very) unstructured standard output. If I had to guess that is actually what the pearl script do...?

**Comment by reskejak** [comment]
Yes, I see that now! It makes me wonder if the HMMER EBI webtool is also running pfam_scan.pl under the hood, even though it's titled "hmmscan".
